File: python/level1/1_lotto.py
#로또의 최고 순위와 최저 순위
#https://programmers.co.kr/learn/courses/30/lessons/77484

def solution(lottos, win_nums):
    answer = []
    count = 0
    zero = 0
    for i in lottos:
        if (i in win_nums):
            count += 1
        if (i == 0):
            zero += 1
    if (count + zero == 0):
        answer.append(6)
    else:
        answer.append(7 - count - zero)
    if (count == 0):
        answer.append(6)
    else:
        answer.append(7 - count)

    return answer

File: java/level1/1_lotto.java
class Solution {
    public int[] solution(int[] lottos, int[] win_nums) {
        int[] answer = {0,0};
        int zero_cnt = 0;
        int cnt = 0;
        for (int i = 0; i<lottos.length; i++) {
            //System.out.println(i);
            if (lottos[i] == 0) {
                zero_cnt++;
            }
            else {
                for (int j = 0; j<win_nums.length; j++){
                    if (lottos[i] == win_nums[j]) {
                        cnt++;
                        break;
                    }
                }
            }
        }
        //System.out.println(cnt);
        if (cnt == 0) {
            answer[1] = 6;
        }
        else 
            answer[1] = 7-cnt;
        if (cnt+zero_cnt == 0) {
            answer[0] = 6;
        }
        else 
            answer[0] = 7-cnt-zero_cnt;
        return answer;
    }
}